app/controllers/chef_ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ChefIngredientsController < ApplicationController
+  def index
+    @chef = Chef.find(params[:id])
+    @ingredients = @chef.ingredients
```

Comment:
This might leave some duplicate ingredients in the list, so we were looking for a `.distinct` to be included. You might also opt to use a `pluck` to just grab the ingredient names. Taken together, this functionality is better off living in your Chef model.